src/3rd_party_repos.c

```diff
@@ -526,4 +526,52 @@ void third_party_repo_header(const char *repo_name)
 	free_string(&header);
 }
 
+static bool is_binary(const char *filename)
+{
+	if (strncmp(filename, "/bin", strlen("/bin")) == 0 ||
```

Comment:
Keep an array of PATHs and check each one in the array instead of hardcoding that.

Reply:
done